certified-connectors/Africa's Talking SMS/apiDefinition.swagger.json

```diff
@@ -210,6 +210,265 @@
+              "description": "from"
+            },
+            "id": {
+              "type": "string",
+              "description": "id"
+            },
+            "linkId": {
+              "type": "string",
+              "description": "linkId"
+            },
+            "text": {
+              "type": "string",
+              "description": "text"
+            },
+            "to": {
+              "type": "string",
+              "description": "to"
+            },
+            "networkCode": {
+              "type": "string",
+              "description": "networkCode"
+            }
+          }
+        },
+        "description": "A new message in your inbox"
+      },
+      "post": {
+        "responses": {
+          "200": {
+            "description": "200"
+          }
+        },
+        "summary": "When you receive a message in your inbox",
+        "description": "Triggered whenever a message is sent to any of your registered shortcodes",
+        "operationId": "InboxNotification",
+        "x-ms-visibility": "important",
+        "x-ms-trigger": "single",
+        "parameters": [
+          {
+            "name": "username",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Your application's username",
+            "x-ms-summary": "Username",
+            "x-ms-visibility": "important",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "name": "Content-Type",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "url": {
+                  "type": "string",
+                  "description": "url",
+                  "title": "",
+                  "x-ms-visibility": "internal",
+                  "x-ms-notification-url": true
+                }
+              },
+              "x-ms-visibility": "internal",
+              "required": [
+                "url"
+              ]
+            },
+            "x-ms-visibility": "internal"
+          }
+        ]
+      }
+    },
+    "/api/v1/powerapps/{username}/sms/optout/callback": {
+      "x-ms-notification-content": {
+        "description": "A bulk sms opt out notification",
+        "schema": {
+          "type": "object",
+          "properties": {
+            "senderId": {
+              "type": "string",
+              "description": "senderId"
+            },
+            "phoneNumber": {
+              "type": "string",
+              "description": "phoneNumber"
+            }
+          }
+        }
+      },
+      "post": {
+        "responses": {
+          "200": {
+            "description": "200"
```

Comment:
@crispinkoech Can you please provide a valid description for the "200" response code instead of "200"